src/app/core/common-components/entity-select/entity-select.component.ts

```diff
@@ -68,61 +73,70 @@ export class EntitySelectComponent<E extends Entity> implements OnChanges {
     this.loadAvailableEntities(type);
   }
 
+  /**
+   * Whether users can select multiple entities.
+   */
+  @Input() multi: boolean = true;
+
   /**
    * The (initial) selection. Can be used in combination with {@link selectionChange}
    * to enable two-way binding to an array of strings corresponding to the id's of the entities.
-   * @param sel The initial selection
+   * @param sel The initial selection (single id string for multi=false, array of id strings for multi=true)
    */
-  @Input() set selection(sel: string[]) {
-    if (!Array.isArray(sel)) {
-      this.selectedEntities = [];
-      return;
-    }
+  @Input() set selection(sel: T) {
+    const selArray = Array.isArray(sel) ? sel : [sel];
```

Comment:
There might be an issue when passing `undefined` which will result in `[undefined]` which might not result in the correct behavior.